runtime/src/bank.rs

```diff
@@ -822,6 +856,12 @@ impl Bank {
 
         if *hash == Hash::default() {
             // finish up any deferred changes to account state
+
+            // DANGER: Remove this guard after tds has transitioned to the eager rent
+            // collection and ABSOLUTELY before the mainnet-beta transitions to v1.1.
+            if !(self.operating_mode() == OperatingMode::Preview && self.epoch() < 47) {
```

Comment:
This is rescheduled:  https://github.com/solana-labs/solana/pull/10028#discussion_r425205226